index.js

```diff
@@ -171,11 +205,20 @@ module.exports = {
       const recps = content.recps
       if (!recps || !Array.isArray(recps) || recps.length < 1)
         return cb(new Error('Missing recps'))
+      const groupId = recps[0]
```

Comment:
we should check more - for this the recps length should be EXACTLY 1, and the should be a URI that `startsWith('ssb:identity/group/')` or whatever we settled on

Reply:
there's this issue for that https://github.com/ssbc/ssb-tribes2/issues/27 which is probably good to do soon/maybe even before this pr so we know we don't break stuff